go.mod

```diff
@@ -264,6 +264,8 @@ require (
 	software.sslmate.com/src/go-pkcs12 v0.6.0
 )
 
+require github.com/sirupsen/logrus v1.9.3 // indirect
```

Comment:
I wonder if we can fix the automatic go.mod changes if we get rid of all the `// replaced` comments so that there's exactly one block of requires with the direct dependencies and no comments and exactly one block of requires with indirect dependencies and `// indirect` comments.